main/test/flix/Test.Exp.Jvm.Object.flix

```diff
@@ -0,0 +1,7 @@
+namespace Test/Exp/Jvm/Object {
```

Comment:
This file should probably be called `...Jvm.NewObject.flix` and its namespace updated.